jscomp/bin/bspp.ml

```diff
@@ -2353,8 +2358,10 @@ let rec default_error_reporter ppf ({loc; msg; sub; if_highlight} as err) =
   if highlighted then
     Format.pp_print_string ppf if_highlight
   else begin
-    fprintf ppf "%a%a %s" print loc print_error_prefix () msg;
-    List.iter (Format.fprintf ppf "@\n@[<2>%a@]" default_error_reporter) sub
```

Comment:
hi, would you update your ocaml submodule? I cherry picked a fix from upstream, thanks